packages/mobile/src/fiatExchanges/ProviderOptionsScreen.test.tsx

```diff
@@ -118,7 +118,7 @@ describe('ProviderOptionsScreen', () => {
     })
   })
 
-  it('opens Transak correctly', async () => {
+  xit('opens Transak correctly', async () => {
```

Comment:
Is skipping intentional here?

Reply:
I think it is because the code that opens Transak was commented out.